ltex/ltex.sh

```diff
@@ -38,13 +38,15 @@ for file in "$@"; do
   fi
 done
 
-echo "${files_to_process[@]}" |
+output=$(echo "${files_to_process[@]}" |
   xargs docker run --rm -i \
     --user "$UID" \
     --volume "$(pwd):/src:rw,Z" \
     --workdir /src \
-    "$ltex_version" |
-  sed -E 's|(app/javascript/documentation/.*?\.md):|\1x:|g' || true
+    "$ltex_version" 2>&1 |
+  sed -E 's|(app/javascript/documentation/.*?\.md):|\1x:|g' || true)
+
+echo "$output" | grep "SEVERE: java.io.IOException: Stream closed" >/dev/null || echo "$output"
```

Comment:
💭 it could make it harder to debug if a future issue would trigger a java IOException that would be stripped from the output?